src/content/getting-started/architecture/globalnet/_index.en.md

```diff
@@ -56,18 +69,18 @@ It mainly consists of two key components: the IP Address Manager and Globalnet.
 The IP Address Manager (IPAM) component does the following:
 
 * Creates a pool of IP addresses based on the `GlobalCIDR` configured on cluster.
-* Allocates a globalIp from the GlobalIp pool on creation of a Pod, Service, or ServiceExport.
-* Annotates the Pod or exported Service with `submariner.io/globalIp=<global-ip>`.
-* On deletion of a Pod, Service, or ServiceExport, releases its globalIp back to the pool.
+* Allocates a globalIp from the GlobalIp pool on creation of ClusterGlobalEgressIPs, GlobalEgressIPs or GlobalIngressIPs.
+* On deletion of a ClusterGlobalEgressIPs, GlobalEgressIPs or GlobalIngressIPs, releases its globalIp back to the pool.
```

Comment:
```suggestion
* On deletion of a ClusterGlobalEgressIPs, GlobalEgressIPs or GlobalIngressIPs, releases its globalIp back to the pool.
```

I'd suggest mirroring the structure of the addition point above for the deletion point, to make it easier to quickly scan/read.

Reply:
This bullet isn't needed with my suggestion above.